### PR TITLE
Adding stereo_slam to documentation index for distro

### PR DESCRIPTION
**Pull request by plnegre**

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -9958,6 +9958,11 @@ repositories:
       type: git
       url: https://github.com/wunderkammer-laboratory/steered_wheel_base_controller.git
       version: master
+  stereo_slam:
+    doc:
+      type: git
+      url: https://github.com/srv/stereo_slam
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: Must end with `.git`.

Same below.

**Comment by dirk-thomas** [comment]
It looks like you recently changed the branch name which lets the Jenkins jobs fail: http://build.ros.org/job/Jdoc__stereo_slam__ubuntu_trusty_amd64/12/console Please fix the version entries.

**Comment by plnegre** [comment]
I don't know what is wrong... The value of version tag on 'ros/distro/indigo/distribution.yaml' is 'indigo' and, on my repo (https://github.com/srv/stereo_slam.git) I have a branch with this name.

**Comment by tfoote** [comment]
The problem is in the Jade entry which references `indigo-devel` branch in `jade/distribution.yaml`

**Pull request merged**